### ISSUE TITLE
Using active objects alias causes error ORA-00904 invalid identifier

### ISSUE THREAD

**Issue by amjmhs** [codescape/jira-scrum-poker#90]
We tried to use Jira Scrum Poker 4.7.0 with Jira 8.5.1 and an Oracle database. At the logfiles we get following error messages:
```
java.sql.SQLSyntaxErrorException: ORA-00904: "sps"."CREATE_DATE": invalid identifier`
`Caused by: Error : 904, Position : 361, Sql = SELECT * FROM ( SELECT sps."CONFIRMED_DATE",sps."ISSUE_KEY",sps."CANCELLED",sps."CONFIRMED_USER_KEY",sps."CONFIRMED_VOTE",sps."CREATE_DATE",sps."VISIBLE",sps."CREATOR_USER_KEY" FROM "AO_1FA2A8_SCRUM_POKER_SESSION" sps JOIN "AO_1FA2A8_SCRUM_POKER_VOTE" spv ON spv."SESSION_ID" = sps."ISSUE_KEY" WHERE spv."USER_KEY" = :1 and sps."CONFIRMED_VOTE" = :2 ORDER BY "sps"."CREATE_DATE" DESC ) WHERE ROWNUM <= 3, OriginalSql = SELECT * FROM ( SELECT sps."CONFIRMED_DATE",sps."ISSUE_KEY",sps."CANCELLED",sps."CONFIRMED_USER_KEY",sps."CONFIRMED_VOTE",sps."CREATE_DATE",sps."VISIBLE",sps."CREATOR_USER_KEY" FROM "AO_1FA2A8_SCRUM_POKER_SESSION" sps JOIN "AO_1FA2A8_SCRUM_POKER_VOTE" spv ON spv."SESSION_ID" = sps."ISSUE_KEY" WHERE spv."USER_KEY" = ? and sps."CONFIRMED_VOTE" = ? ORDER BY "sps"."CREATE_DATE" DESC ) WHERE ROWNUM <= 3, Error Msg = ORA-00904: "sps"."CREATE_DATE": invalid identifier`
```

This looks a lot like [this issue](https://community.atlassian.com/t5/Jira-questions/Using-active-objects-alias-causes-error-ORA-00904-invalid/qaq-p/677687).

**Comment by codescape** [comment]
Hello @amjmhs,

thank you very much for the issue and your PR! I just merged it into the master branch and will release it as soon as possible.

**Comment by codescape** [comment]
This issue is solved with Scrum Poker for Jira 4.8.1
